assume/reinforcement_learning/learning_role.py

```diff
@@ -232,8 +237,14 @@ def compare_and_save_policies(self) -> None:
 
         Notes:
             This method is typically used during the evaluation phase to save policies that achieve superior performance.
+            Curretnly the best evaluation metrik is still assessed by the development team and preliminary we use the avergae rewards.
         """
-        modes = ["reward", "profit", "regret"]
+        # modes = ["reward", "profit", "regret"]
+        modes = ["reward"]
+
+        # add current rewrad to list of all rewards
+        self.rl_eval_rewards.append(current_avg_reward)
+
         for mode in modes:
```

Comment:
do we even need this for loop now if we are using only a single mode?

Reply:
I inteded to keep the variability for the future but make it as easy as possible and still have the average reward as the target. But Folrian and I made it more generic and still have the avg reward.